tests/framework/utils/snapshot.go

```diff
@@ -27,7 +27,7 @@ import (
 const (
 	// snapshotterVersion from which the snapshotcontroller and CRD will be
 	// installed
-	snapshotterVersion = "v4.0.0"
+	snapshotterVersion = "v5.0.1"
```

Comment:
earlier snapshotter was getting deployed in the default namespace, after 4.0.0 its getting deployed in kube-system namespace, I think this change is still not enough we might need to do a string replace to replace kube-system with default or change https://github.com/rook/rook/blob/c57416b2614dc8fa9d2b34ab7ac718937edd56f6/tests/framework/utils/snapshot.go#L86 to kube-system

Reply:
@sathieu as you are away and we need this fix on priority I send https://github.com/rook/rook/pull/9665 added you as co-author.